includes/emails/class-give-email-tags.php

```diff
@@ -726,7 +726,7 @@ function give_email_tag_date( $tag_args ) {
 
 	switch ( true ) {
 		case give_check_variable( $tag_args, 'isset', 0, 'payment_id' ):
-			$date = date_i18n( give_date_format(), strtotime( get_the_date( $tag_args['payment_id'] ) ) );
+			$date = date_i18n( give_date_format(), strtotime( get_the_date( '', $tag_args['payment_id'] ) ) );
```

Comment:
Rather than passing empty here couldn't you pass `U` and get the unix timestamp and remove the `strtotime()` call? That would be more effecient. See: https://codex.wordpress.org/Formatting_Date_and_Time

Reply:
Thanks @DevinWalker what was really nice 

Updated the PR